test/integration/trailing-slashes/next.config.js

```diff
@@ -2,4 +2,5 @@ module.exports = {
   experimental: {
     // <placeholder>
   },
+  // basePath: '/docs',
```

Comment:
Was this comment out intentional?

Reply:
Yes, the tests weren't actually testing with a basepath because of this comment missing